Arcs/Arcs.cpp

```diff
@@ -60,12 +74,18 @@ PROGRAM " " PACKAGE_VERSION "\n"
 "   -m, --index_multiplicity=RANGE  barcode multiplicity range [50-10000]\n"
 "   -d, --max_degree=N    max node degree in scaffold graph [0]\n"
 "   -e, --end_length=N    contig head/tail length for masking alignments [30000]\n"
-"   -r, --error_percent=N p-value for head/tail assignment and link orientation\n"
-"                         (lower is more stringent) [0.05]\n"
+"   -r, --error_percent=N p-value for head/tail assignment and link orientation (lower is more stringent) [0.05]\n"
 "   -v, --run_verbose     verbose logging\n"
+"ARCS Specific Options:\n"
+"   -f, --file=FILE       FASTA file of contig sequences to scaffold [optional]\n"
+"   -s, --seq_id=N        (only in default mode)min sequence identity for read alignments [98]\n"
+"ARKS Specific Options:\n"
+"   -f, --file=FILE       FASTA file of contig sequences to scaffold [required]\n"
```

Comment:
Let's just move this to the common options -- seems redundant to have it under ARKS and ARCS when it has the same meaning.

Reply:
I put them separately because it is optional for ARCS and required for ARKS. Now I think it is better to delete it from ARKS options as it is Required and shown in "Usage:' explicitly  and just keep it as option for ARCS. But I can move it to "Common Options" as well, if you think it is a better idea.

Reply:
Let's keep it in the usage, like you say, but also move to 'common options'. I think better to have it in multiple places. I know it is technically optional for ARCS, but I'd rather users err-ed on the side of specifying it as a sanity check